### PR TITLE
OCPQE-18643: Add python3 deps required to run pruning

### PR DESCRIPTION
**Pull request by jadhaj**
The PR addresses jira ticket https://issues.redhat.com/browse/OCPQE-18643

### CODE REVIEW

File: images/fcos-bastion-image/Containerfile [hunk 1]
```diff
@@ -3,4 +3,4 @@ FROM quay.io/openshifttest/fcos-bm:base
 COPY root/ /
 
 RUN set -x; pip3 install --target=/usr/lib redfish pandas \
-    && systemctl preset-all
+    && pip3 install netaddr junos-eznc wheel && systemctl preset-all
```
Comment (aleskandro) [inline images/fcos-bastion-image/Containerfile:6]: Hi Jad, add the three deps one line before together with redfish and pandas. Also, let's ensure the build (`images` job) succeeds.

File: images/fcos-bastion-image/Containerfile [hunk 2]
```diff
@@ -2,5 +2,5 @@ FROM quay.io/openshifttest/fcos-bm:base
 
 COPY root/ /
 
-RUN set -x; pip3 install --target=/usr/lib redfish pandas \
+RUN set -x; pip3 install --target=/usr/lib redfish pandas netaddr wheel git+https://github.com/jadhaj/py-junos-eznc.git@fix \
```
Comment (aleskandro) [inline images/fcos-bastion-image/Containerfile:5]: mmm looking at your fix there, it seems we should wait for https://github.com/Juniper/py-junos-eznc/pull/1281/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552 to have support for python4.12.

Comment (jadhaj) [reply images/fcos-bastion-image/Containerfile:5]: yes. looks like

Comment (aleskandro) [reply images/fcos-bastion-image/Containerfile:5]: I'm unsure, but to unblock our job, we could try installing from either

https://github.com/Juniper/py-junos-eznc/pull/1281

or https://github.com/Juniper/py-junos-eznc/pull/1288

by pulling those branches into your personal fork temporarily.

Comment (jadhaj) [reply images/fcos-bastion-image/Containerfile]: PR https://github.com/Juniper/py-junos-eznc/pull/1281 was merged and closed

**Comment by jadhaj** [comment]
@aleskandro ci/prow/shellcheck  failed and complaining about X prefix 
i did not complain about before.

**Comment by aleskandro** [comment]
Hi @jadhaj , can you rebase to doublecheck shellcheck? You shouldn't care about a script you didn't modify.

**Comment by jadhaj** [comment]
> Hi @jadhaj , can you rebase to doublecheck shellcheck? You shouldn't care about a script you didn't modify.

shellcheck passed after rebase

**Comment by aleskandro** [comment]
Great, can you squash the commits into one and test on arm too?

**Comment by jadhaj** [comment]
tested on arm:
https://privatebin.corp.redhat.com/?be12e5125186adf6#9m9R58U4SNoEhxUrSFBtNPaabvmye5zmPsURJYMnuyGK

**Comment by aleskandro** [comment]
/lgtm
/approve

Thanks Jad!

**Pull request merged**